.travis.yml

```diff
@@ -5,5 +5,8 @@ branches:
 go: "1.12"
 env:
 - GO111MODULE=on
-# NOTE: `go fmt` does not work, see https://go-review.googlesource.com/c/go/+/153459/
-script: go build cmd/remediator/app.go && [ -z "`ls */**/*.go | xargs -n1 go fmt`" ]
+script: |
+  set -ex
+  go build -o .build/remediator cmd/remediator/app.go
+  [ -z "`ls */**/*.go | xargs -n1 go fmt`" ] # `go fmt` does not work, see https://go-review.googlesource.com/c/go/+/153459/
```

Comment:
But the issue was fixed in Dec 2018 in 1.12. https://github.com/golang/go/issues/28011

Reply:
🤷‍♂ 
```
go fmt
can't load package: package github.com/aksgithub/kube_remediator: unknown import path "github.com/aksgithub/kube_remediator": cannot find module providing package github.com/aksgithub/kube_remediator
```

Reply:
`go fmt ./...` works perfectly.

Reply:
updated